### DISCUSSION TITLE
SoftGPU on PCbox Windows 9x VMs would make this emulator the GOAT.

### DISCUSSION THREAD

**Discussion by Mrtechenthuiast** [PCBox/PCBox#99] [General]
Hello my name is Andrew Schmitt, I like where you are going with the PCbox emulator with Pentium III emulation and and upcoming Pentium IV emulation and that's great. Pardon me for asking you this but, have you ever considered having PCbox interfacing with the host machine for SoftGPU drivers for PCbox Windows 95 to XP VMs? I mean SoftGPU brought back Windows 9x 3D hardware acceleration on PC emulators in the most recent years on Virtual Box and on VMware VIA passthrough from the Host system. I know it may sound risky, but I am thinking that having PCbox using SoftGPU drivers on PCbox Windows 9x to XP VMs by having passthrough from the Host machine might not be a bad idea.

What do you think?

**Comment by bachimusprime** [comment]
This emulator is more experimentally-focused and the emulation of the later CPUs is extremely slow, even on modern hardware. I wish to preserve old hardware, not just become some one-stop shop for PC emulation.

**Reply by Mrtechenthuiast** [reply]
I understand, please take your time with this Project.
